cylc/flow/workflow_events.py

```diff
@@ -106,9 +106,9 @@ def _run_event_mail(self, config, ctx):
                     'host': ctx.host,
                     'port': ctx.port,
                     'owner': ctx.owner,
-                    'suite': ctx.suite}
+                    'workflow': ctx.workflow}
```

Comment:
I think we want to keep `suite` as a deprecate field here too.

Reply:
Yep. I've also handled bad mail footer templates here with a warning, instead of falling over dead.